database/mssql/scripts/dbscripts/db-upgrade.sh

```diff
@@ -11,6 +11,12 @@ eval $(grep -v '^#' ../../.env | xargs)
 
 echo "===== Begin DB Schema Upgrade ====="
 
+# 1. Find current DB version 
+#    select STATIC_VARIABLE_VALUE from PIMS_STATIC_VARIABLE where STATIC_VARIABLE_NAME = 'dbversion'
+# 2. Search for all SQL scripts under the 'Alter Up' folder and order them by folder name and then file name
+#    find ./PSP*/'Alter Up' -type f -iname "*.sql" | sort -n
+# 3. Execute all SQL scripts that have a higher version than the current DB version
+ 
```

Comment:
This section is already present in lines 3-10 above - any reason to duplicate here?